test/functional/apps/dashboard/index.js

```diff
@@ -63,14 +63,14 @@ export default function ({ getService, loadTestFile, getPageObjects }) {
     describe('using legacy data', function () {
       before(() => remote.setWindowSize(1200, 900));
 
-      loadTestFile(require.resolve('./_dashboard_time_picker'));
-      loadTestFile(require.resolve('./_bwc_shared_urls'));
-      loadTestFile(require.resolve('./_panel_controls'));
-      loadTestFile(require.resolve('./_dashboard_state'));
-      loadTestFile(require.resolve('./_dashboard_save'));
+      // loadTestFile(require.resolve('./_dashboard_time_picker'));
+      // loadTestFile(require.resolve('./_bwc_shared_urls'));
+      // loadTestFile(require.resolve('./_panel_controls'));
+      // loadTestFile(require.resolve('./_dashboard_state'));
+      // loadTestFile(require.resolve('./_dashboard_save'));
       loadTestFile(require.resolve('./_dashboard_time'));
```

Comment:
If you copy/paste this line like 20x you can get more feedback quickly.

Reply:
Good idea!